tutorials/4 Analysis/B. Advanced - Direct use of the renderers/4.14 Analyze a double hanger resonator (S Param).ipynb

```diff
@@ -278,24 +261,22 @@
     "\n",
     "#### Setup\n",
     "\n",
-    "Select the analysis you intend to run from the `qiskit_metal.analyses` collection.<br>\n",
+    "Select the analysis you intend to run from the `qiskit_metal.analyses.simulation` collection.<br>\n",
```

Comment:
this comment should actually remain qiskit_metal.analyses (As in.....everywhere in that folder, including sub-folders). But given this notebook only runs the simulation stuff, we can let this pass.